APIAssignment/src/test/java/APIAssignments/Product.java

```diff
@@ -0,0 +1,84 @@
+package APIAssignments;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Method;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.logging.*;
+
+import static io.restassured.RestAssured.baseURI;
+import static io.restassured.RestAssured.given;
+
+public class Product {
+    private static String url;
+
+    {
+        try {
+            url = ConfigReader.getUrl();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private Logger logger = Logger.getLogger("Product.class");
+
+    @Test(priority = 1)
+    public void validateProductList() {
+        baseURI = url;
+        var getALLProductList = given().when().get(baseURI).then().log().all().toString();
+        logger.log(Level.INFO, "List return by API is " + getALLProductList);
+    }
+
+    @Test(priority = 2)
+    public void validateStatusCode() {
+        baseURI = url;
+        logger.info("getAllProductsList baseURI:" + baseURI);
+        RequestSpecification httpRequest = given();
+        Response response = httpRequest.request(Method.GET);
```

Comment:
If there is a path parameter or query parameter you can add it here

Reply:
DONE !